test/is.allow.token.test.ts

```diff
@@ -0,0 +1,25 @@
+import { RelayingServices } from '../src';
+import { MockRelayingServices } from './mock';
+import Expect = jest.Expect;
+import {
+    MOCK_TOKEN_ADDRESS
+} from './constants';
+
+declare const expect: Expect;
+
+describe('Tests for is allow token', () => {
+    let sdk: RelayingServices;
+
+    beforeEach(async () => {
+        sdk = new MockRelayingServices();
+        sdk.initialize({});
+    });
+
+    it('Should run is allow token', async () => {
+        try{
+            await sdk.isAllowedToken(MOCK_TOKEN_ADDRESS);
```

Comment:
As pointed out here: https://github.com/infuy/relaying-services-sdk/pull/2/files#r701722107

Reply:
It looks like it isn't resolved yet